### ISSUE TITLE
can't encode Ext G CJKV Character into GB18030

### ISSUE THREAD

**Issue by Clayblockunova** [gchq/CyberChef#1946]
**Describe the bug**
can't encode Ext G CJKV Character into GB18030

**To Reproduce**
1. Choose "Encode text-Simplified Chinese GB18030";
2. Type Ext G CJKV Character (like"𰋸") in input;
3. Empty bytes would be returned.

**Expected behaviour**
Ext G CJKV Character should be encoded into corresponding GB18030 bytes.

**Comment by kryptokazz** [comment]
Have you checked that the output matches the correct encoding type? 
https://github.com/gchq/CyberChef/issues/1646

**Comment by Clayblockunova** [comment]
> Have you checked that the output matches the correct encoding type? #1646

yes. additionaly, when converting with "input character encoding" option, non-GBK Character  also become Empty bytes (1 per 4 UTF16 bytes).
